packages/sdk/python/tests/test_writeback_consumer.py

```
@@ -0,0 +1,95 @@
+
+
+class FakeClient:
+    def __init__(self, items: list[WritebackItem]) -> None:
+        self.items = items
+        self.acks: list[Any] = []
+
+    def list_pending_writebacks(self, workspace_id: str) -> list[WritebackItem]:
+        assert workspace_id == "ws_123"
+        return self.items
+
+    def ack_writeback(self, input: Any) -> dict[str, Any]:
```

Ruff (0.15.18)

[error] Function argument `input` is shadowing a Python builtin

(A002)